tasks/create-advisory/create-advisory.yaml

```diff
@@ -81,14 +81,15 @@ spec:
         #
         # detect which one to use based on repositories specified
         #
-        pending_repositories=$(jq -r '.mapping.components[] | select(.repository |
+        # Updated repository checks using snapshot
```

Comment:
What's this comment supposed to mean?

Reply:
I just wanted to highlight that the repository value is now fetched from snapshot JSON, not data JSON.

Reply:
Imagine reading the comment once this PR is merged. Does the code update something? The update happens in this commit. So it doesn't make sense unless I'm misunderstanding what "update" means here.

Reply:
Yes, I got your point.

Reply:
It was fixed, and the comment can be resolved. Thanks.